data/abilities/ba_pa_med.json

```diff
@@ -1,5 +1,7 @@
 {
     "name": "Attack",
+    "description": "Medium physical attack",
     "power": 10,
-    "type": "physical"
+    "type": "physical",
+    "jp_cost": 100
```

Comment:
Any reason to not just let this use the default `jp_cost`? Basic attacks are never purchased, so giving them an explicit cost seems weird.

Reply:
I was torn on just leaving it alone for both the reason it was 100, and since it's not purchased.

The only reason I did include was to show that the formula between melee and ranged resulted in the same JP value. I found that interesting, and exciting. I am fine with removing and so I will.